utils/cryptutils/cryptutils.go

```diff
@@ -260,6 +260,13 @@ func (cryptoContext *CryptoContext) GetClientTLSConfig() (*tls.Config, error) {
 	return &tls.Config{RootCAs: cryptoContext.rootCertPool, MinVersion: tls.VersionTLS12}, nil
 }
 
+// CertToPEM is a utility function returns a PEM encoded x509 Certificate.
+func CertToPEM(cert *x509.Certificate) []byte {
```

Comment:
You can use `pem.EncodeToMemory` in place?

Reply:
Decided to leave it as is.